templates/admin/players.html.twig

```diff
@@ -1,5 +1,6 @@
 {% extends '@EasyAdmin/default/layout.html.twig' %}
 {% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}"/>
```

Comment:
As far as I understand, it's enough to include js file, css file is already included in tournament.js. Please try removing this link. Stylesheets should be included in <head> not in <body> tag